api/src/stampy_chat/env.py

```diff
@@ -15,6 +15,8 @@
 LOGGING_URL      = os.environ.get('LOGGING_URL')
 PINECONE_INDEX   = None
 
+FLASK_PORT = int(os.environ.get('FLASK_PORT', '3001'))
```

Comment:
otherwise there's a port conflict, as both the frontend and backend use the same port